cs/mailing.texy

```diff
@@ -163,10 +163,23 @@ V šabloně pak odkaz vytvoříme pomocí makra `{link}` tak, jak jsme zvyklí.
 <a href="{link Presenter:action}">Odkaz</a>
 \--
 
-Odesílání přes SMTP
-===================
+Typy mailerů
+============
 
-Výchozí mailer((třída zajišťující odesílání e-mailů)) používá PHP funkci [php:mail]. Pokud bychom potřebovali poštu odesílat skrze SMTP server, použijte `SmtpMailer`
+|* Mailer        |* Podpora SMTP
+| SendmailMailer | Ne
+| SmtpMailer     | Ano
+| FallbackMailer | Ano
```

Comment:
bylo to myšleno tak, že pomocí SmtpMailer má podporu.
Nejspíš nechat jenom seznam s odkazy na api ?